docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -0,0 +1,80 @@
+
+Getting Started with Galaxy
+===========================
+
+Tutorial Goals
+--------------
+
+-  Learn about `Galaxy <https://training.galaxyproject.org/>`__
+-  Create and run a basic Galaxy workflow
+-  Export the workflow to a file
+-  Setup a GitHub account and repository
+-  Push the workflow to GitHub
+-  Make a GitHub release
+
+Create a basic Galaxy workflow
+------------------------------
+
+Create and run your workflow in Galaxy. Here  is a tutorial for `creating, editing, running and importing Galaxy workflows <https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html>`__
+
+Export the workflow to a file
+-----------------------------
+
+- Click on the Galaxy UI Workflow link at the top of the page.
+- Click on the workflow name to expose the drop down menu.
+- Click Download
+
+The exported YAML (Or JSON) file with a '.ga' suffix describes the inputs,
+outputs, and Galaxy Tool Shed dependencies for your workflow.
+
+
+.. figure:: /assets/images/docs/galaxy_download.png
+   :alt: Download
+
+   Download
+
+Setting up GitHub
+-----------------
+
+You will need to add the Galaxy workflow file you downloaded to a source code
+repository that Dockstore knows about. GitHub is a good choice, and if you
+are not familiar with GitHub you can use this
+`tutorial <https://guides.github.com/activities/hello-world/>`__ to set up
+an account and repository.
+
+Upload the workflow to GitHub
+-----------------------------
+
+- Go to your repository and click on the Upload Files menu item under Add Files
+- Click on the 'choose your files' link
+- Select your '.ga' Galaxy workflow file
+- Click on 'Commit changes'
+
+These steps are outlined `here <https://docs.github.com/en/github/managing-files-in-a-repository/adding-a-file-to-a-repository>`__
+
+Releasing on GitHub
+-------------------
+
+Now that we've successfully created our workflow in Galaxy and tested it the
+workflow is ready to share with others. Making a release on GitHub will tag
+your GitHub repository with a version tag so you can always get back to
+this particular release. Follow the steps outlined `here <https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository>`__
+to create a release.
+
+Next Steps
+----------
+
+Now that you have a git repository that includes a Galaxy workflow, and you
+have tested it and are satisfied that it works the next step is to
+register it on Dockstore.
+
+Follow the :doc:`next tutorial <register-on-dockstore/>` to create an
+account on Dockstore and link third party services.
```

Comment:
Linking 3rd party services is not really necessary to register a Galaxy, unless you count GitHub, but you typically get that for free. More importantly, I think you should have another link to registering a workflow (the page you link to, has as its next step how to register a tool, which doesn't apply to Galaxy), and more particularly, since this is new, I think you should direct them to registering with GitHub apps.

Reply:
Added an extra link pointing to workflow registration using GitHub apps.